### PR TITLE
Add uuid field

### PR DESCRIPTION
**Pull request by mjkaufer**
Simple PR to fix https://github.com/HackGT/registration/issues/169

### CODE REVIEW

**Comment by petschekr** [comment]
We need this to apply retroactively as well. Also, what are the benefits of the `uuid` library over just a `crypto.randomBytes()` call?

**Comment by mjkaufer** [comment]
All the cool kids use the `uuid` library. Also, I figured the move is to migrate this stuff by hand, so we don't have migration code in the repo that'll only be used once

File: server/schema.ts [hunk 1]
```diff
@@ -195,7 +198,8 @@ export const User = mongoose.model<IUserMongoose>("User", new mongoose.Schema({
 	confirmationStartTime: Date,
 	confirmationSubmitTime: Date,
 
-	admin: Boolean
+	admin: Boolean,
+	uuid: String
```
Comment (petschekr) [inline server/schema.ts]: Please set this to be a required field for new documents

File: server/schema.ts [hunk 2]
```diff
@@ -137,6 +137,9 @@ export interface IUser {
 
 	admin?: boolean;
 
+	uuid?: string;
```
Comment (petschekr) [inline server/schema.ts]: We should update existing users to include a uuid when we merge this so please make this non-optional

**Comment by illegalprime** [comment]
Please rebase on top of `graphql-v2` and make the graphql api use only UUIDs.

**Pull request merged**